DigitalLearningSolutions.Web/Styles/trackingSystem/centreAdministrators.scss

```diff
@@ -1,5 +1,6 @@
-﻿@import "~nhsuk-frontend/packages/core/all";
-@import "../shared/searchableElements/searchableElements";
+﻿@use "nhsuk-frontend/packages/core/all" as *;
+@use "../shared/searchableElements/searchableElements" as *;
+@use "../shared/breakpoints" as *;

```

Comment:
Breakpoints just defines a variable for the xl-desktop width in pixels so that it can be used in certain screen-size conditional styles. Previously, it would have been imported from searchableElements imports, but now we need to explicitly declare it. Think there are a couple more places like this as well.